examples_tests/60.ClusteredRendering/debug_draw_aabb.vert

```diff
@@ -0,0 +1,28 @@
+#version 460 core
+
+#include <nbl/builtin/glsl/utils/common.glsl>
+#include <nbl/builtin/glsl/utils/transform.glsl>
+#include <nbl/builtin/glsl/shapes/aabb.glsl>
+
+layout (set = 0, binding = 0) restrict buffer readonly DebugAABBs
+{
+	nbl_glsl_shapes_AABB_t data[];
+} debugAABBs;
+
+layout (set = 0, binding = 1, row_major, std140) uniform UBO
+{
+    nbl_glsl_SBasicViewParameters params;
+} CamData;
+
+layout(location = 0) out vec3 color;
+
+void main()
+{
+	const nbl_glsl_shapes_AABB_t aabb = debugAABBs.data[gl_InstanceIndex];
+
+	const bvec3 mask = bvec3(gl_VertexIndex&0x1u,gl_VertexIndex&0x2u,gl_VertexIndex&0x4u);
+	vec3 pos = mix(aabb.minVx,aabb.maxVx,mask);
+
+	gl_Position = nbl_glsl_pseudoMul4x4with3x1(CamData.params.MVP, pos);
+	color = vec3(1.f, 0.f, 0.f);
+}
```

Comment:
can you add this shader to `nbl/builtin/glsl/ext/DebugDraw/`

Also move those there while you're at it
https://github.com/Devsh-Graphics-Programming/Nabla/blob/124355408c8a96428205f9decb2c7eb6c75a4073/include/nbl/ext/DebugDraw/Draw3DLineShaders.h

Reply:
@nipunG314 make this but in HLSL, also add an AABB struct to HLSL with some methods

```cpp
namespace nbl::hlsl::shapes // just a pseudocode shorthand, DXC doesn't like nested namespaces
{
template<typename CoordT>
struct AABB
{
    CoordT Min,Max;
};
}
```

look at the old GLSL `nbl_glsl_shapes_AABB_` prefixed functions and `core::aabbox3d` for ideas about what methods would be useful for that class.